website/docs/integrations/github-actions/atmos-terraform-apply.md

```diff
@@ -37,51 +37,62 @@ We recommend combining this action with the [`affected-stacks`](/integrations/gi
 
 :::
 
+The action expects the atmos configuration file `atmos.yaml` to be present in the repository.
+The config should have the following structure:
 
 ```yaml
-  # .github/workflows/atmos-terraform-apply.yaml
-  name: "atmos-terraform-apply"
-
-  on:
-    workflow_dispatch:
-    pull_request:
-      types:
-        - closed
-      branches:
-        - main
-
-  # These permissions are required for GitHub to assume roles in AWS
-  permissions:
-    id-token: write
-    contents: read
-
-  jobs:
-    apply:
-      runs-on: ubuntu-latest
-      steps:
-        - name: Terraform Apply
-          uses: cloudposse/github-action-atmos-terraform-apply@v2
-          with:
-            component: "foobar"
-            stack: "plat-ue2-sandbox"
+# ./rootfs/usr/local/etc/atmos/atmos.yaml
+integrations:
+  github:
+    gitops:
+      terraform-version: 1.5.2
+      infracost-enabled: false
+      artifact-storage:
+        region: us-east-2
+        bucket: cptest-core-ue2-auto-gitops
+        table: cptest-core-ue2-auto-gitops-plan-storage
+        role: arn:aws:iam::xxxxxxxxxxxx:role/cptest-core-ue2-auto-gitops-gha
+      role:
+        plan: arn:aws:iam::yyyyyyyyyyyy:role/cptest-core-gbl-identity-gitops
+        apply: arn:aws:iam::yyyyyyyyyyyy:role/cptest-core-gbl-identity-gitops
+      matrix:
+        sort-by: .stack_slug
+        group-by: .stack_slug | split("-") | [.[0], .[2]] | join("-")
 ```
 
-with the following configuration as an example:
+> [!IMPORTANT]
+> **Please note!** This GitHub Action only works with `atmos >= 1.63.0`. If you are using `atmos < 1.63.0` please use [`v1` version](https://github.com/cloudposse/github-action-atmos-terraform-plan/tree/v1).
```

Comment:
this is all copy and pasted from `affected-stacks.md` above. Rather than having the same steps on both of these pages, can we create a new page for requirements and then link both these 2 pages to that new page?

That page could also be used for this ticket: https://linear.app/cloudposse/issue/DEV-1693/document-atmos-plan-storage-action-on-atmostools

Reply:
Good suggestion @milldr! We should put it here https://github.com/cloudposse/atmos/blob/12b5f67974859302fc3edcf0be738a98050f8c10/website/docs/integrations/github-actions/github-actions.mdx

And refer to it in the other documents.

Reply:
@goruha please see this conversation